### PR TITLE
Blog app - integration specs for views

### PR DESCRIPTION
**Pull request by toliboff**
### In this PR were implemented following:
 - Use Capybara to write integration tests for each view in your project.
 - Add integration tests for Login page
 - Add integration tests for User index page:
 - Add integration tests for User show page
 - Add integration tests for Post index page
 - Add integration tests for Post show page

### CODE REVIEW

**Comment by Davidosky007** [comment]
## Hi @toliboff @m05e5 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that the majority of the contribution is one-sided as pair programming was used for this project you are expected to contribute almost equally 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: spec/views/post_index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+require 'rails_helper'
+RSpec.describe 'Post', type: :system do
```
Comment (Davidosky007) [inline spec/views/post_index_spec.rb]: * I suggest you make Specs file for the integration tests type is `feature` for example ` describe 'the signin process', type: : feature do`

File: spec/views/post_show_spec.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+RSpec.describe 'Post', type: :system do
```
Comment (Davidosky007) [inline spec/views/post_show_spec.rb]: * I suggest you make Specs file for the integration tests type is `feature` for example ` describe 'the signin process', type: : feature do`

File: spec/views/user_index_spec.rb [hunk 3]
```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.describe 'Login page', type: :system do
```
Comment (Davidosky007) [inline spec/views/user_index_spec.rb]: * I suggest you make Specs file for the integration tests type is `feature` for example ` describe 'the signin process', type: : feature do`

File: spec/views/user_show_spec.rb [hunk 4]
```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+RSpec.describe 'User show page', type: :system do
```
Comment (Davidosky007) [inline spec/views/user_show_spec.rb]: * I suggest you make Specs file for the integration tests type is `feature` for example ` describe 'the signin process', type: : feature do`

File: spec/requests/posts_spec.rb [hunk 5]
```diff
@@ -1,49 +1,49 @@
...
+# require 'rails_helper'
+
+# RSpec.describe 'Posts', type: :request do
+#   describe 'GET /index' do
+#     user = User.create(name: 'Tolib', bio: 'Student learning Ruby on Rails')
+#     Post.create(title: 'Post title', text: 'Lorem ipsum dolor set amet...', author: User.first)
+#     before(:example) { get user_posts_path(1) }
+
+#     it 'should have correct response status' do
+#       expect(response).to have_http_status(:ok)
```
Comment (Davidosky007) [inline spec/requests/posts_spec.rb]: * Kindly remove commented code from your codebase as it makes your codebase bulky and unprofessional 🙏

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @toliboff @m05e5 👋, 

Good job so far! 💪 
* Descriptive commit messages ✔️ 
* No linter errors ✔️ 
* Proper use of git-flow ✔️ 
* Well written PR ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AshakaE — APPROVED** [review]
### Hi @toliboff  🙋‍♀️, 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**